src/offsetRotate.js

```diff
@@ -0,0 +1,104 @@
+/* global internalScope */
+
+(function () {
+  function isNumeric (number) {
+    return !isNaN(number);
+  }
+
+  function convertToDegrees (angle) {
```

Comment:
Can this method be in helpers.js, and re-used by the independent 'rotate' property ?